docs/contributing.md

```diff
@@ -15,14 +32,28 @@ You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
+## How to Contribute
+Before proposing changes, look for similar ones in the project's [issues](https://github.com/google/zx/issues) and [pull requests](https://github.com/google/zx/pulls). If you can't decide, create a new [discussion](https://github.com/google/zx/discussions) topic, and we will help you figure it out. When ready to move on:
+* Prepare your development environment.
+  * Ensure you have Node.js 20+ installed.
+  * Bash is essential for running zx scripts. Linux and macOS users usually have it installed by default. Consider using [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install) or [Git Bash](https://git-scm.com/downloads) if you are on Windows.
+* Fork [the repository](https://github.com/google/zx).
+* Create a new branch.
+* Make your changes.
+  * If you are adding a new feature, please include additional tests. The coverage threshold is 98%.
+  * Create a [conventional-commits](https://www.conventionalcommits.org/en/v1.0.0/) compliant messages.
+* Ensure that everything is working:
+  * `npm run fmt` to format your code.
+  * `npm run cov` to run the tests.
```

Comment:
npm run cov  > npm test

Reply:
`npm run cov` not exist. Maybe `npm run test:coverage` ?

Reply:
Thanks!
Changed that